db/migrate/20150107161258_add_admin_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddAdminToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :admin, :boolean, default: false
```

Comment:
You don't have to do it here, but if you were going to do it all over again, it would be worth consider having a `role` column so that you can have multiple types of users (paying, non-paying, lower level admin, higher admin, etc.)